File: pinyougou-web-shop/src/main/webapp/js/controller/homeController.js
app.controller('homeController', function ($scope, $controller,  homeService) {
    $controller("baseController", {$scope: $scope})//继承controller

    //获取面板信息
    $scope.panel=function () {
        homeService.panel().success(function (response) {

            $scope.entity=response;

        })
    }

})

File: pinyougou-web-shop/src/main/java/cn/itcast/core/controller/HomeController.java
package cn.itcast.core.controller;

import cn.itcast.core.service.HomeService;

import com.alibaba.dubbo.config.annotation.Reference;
import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.Map;
@RestController
@RequestMapping("/home")
public class HomeController {

    @Reference
    private HomeService homeService;
    @RequestMapping("/panel")
    public Map<String,Object> panel(){
        String name = SecurityContextHolder.getContext().getAuthentication().getName();

        return homeService.panel(name);
    }

}
